package.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Define the package metadata
+if [ -z "$VERSION" ]; then
+  VERSION='dev'
+fi
+
+project=$APP
+distribution=$DISTRO
+version=$VERSION
+iteration="git$(git rev-parse --short HEAD)"
+architecture='x86_64'
+url="https://github.com/joemiller/kubits"
+description="Kubernetes management functions."
```

Comment:
Please sync this up with the one-liner description from the README: "Collection of utilities to improve working with Kubernetes clusters."